OxfordBodleian/Juel-Jensen/BDLaethe38.xml

```diff
@@ -506,8 +505,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         pencil. Below, another sticker with the signature "Ms 41"printed.</desc>
                                 </item>
                                 <item xml:id="e2">
-                                    <locus target="#ir"/>
-                                    <desc> Writings in pencil: "81 folios. 10 full page paintings. 2 sketches. size 7" x 5"".</desc>
+                                    <desc>Writings in pencil are drawn up on the recto side of the protective bifolium glued to the front cover.
+                                        <q xml:lang="en">81 folios. 10 full page paintings. 2 sketches. size 7" x 5"</q>.</desc>
                                 </item>
                                 <item xml:id="e3">
                                     <note>Aid readings for weekday readings are written in the upper margin within a frame decorated in black and red: 
```

Comment:
Repeating the word "readings" twice in the sentence is not great, perhaps "Aids for identifying weekday readings are written in the upper margin...etc."